resources/pool.rb

```diff
@@ -80,7 +80,7 @@
 property :smp_processor_affinity_mask, Float, default: 4_294_967_295.0, coerce: proc { |v| v.to_f }
 property :smp_processor_affinity_mask_2, Float, default: 4_294_967_295.0, coerce: proc { |v| v.to_f }
 
-property :running, [true, false], desired_state: true
+property :running, [true, false]
```

Comment:
@smurawski that's awesome, definitely not how i read it ;) but i'm glad that the property to make a property private exists, thanks